src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -44,20 +44,21 @@
         "symfony/process": "~2.8|~3.0",
         "symfony/security-core": "~3.2",
         "symfony/security-csrf": "~2.8|~3.0",
-        "symfony/serializer": "~2.8|~3.0",
+        "symfony/serializer": "~3.3",
         "symfony/translation": "~2.8|~3.0",
         "symfony/templating": "~2.8|~3.0",
         "symfony/validator": "~3.2",
         "symfony/yaml": "~3.2",
-        "symfony/property-info": "~2.8|~3.0",
+        "symfony/property-info": "~3.1",
```

Comment:
Because we change the framework's serializer requirement to 3.3, and the serializer in 3.3 requires this. The build with low deps was failing otherwise:

> 1) Symfony\Bundle\FrameworkBundle\Tests\Functional\SerializerTest::testDeserializeArrayOfObject
Error: Class 'phpDocumentor\Reflection\FileReflector' not found

See https://travis-ci.org/symfony/symfony/jobs/191965990